infrastructure/kctf-samples/apache-php/chal.conf

```diff
@@ -1,4 +1,4 @@
 # this file will be sourced by the deployment scripts
 DEPLOY=true
-PUBLIC=false
+PUBLIC=true
```

Comment:
is this intended to be public by default?

Reply:
`git commit -a` hits again